packages/cyberstorm/src/components/Breadcrumb.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { Fragment } from "react";
+
+import { faHouse, faSlashForward } from "@fortawesome/pro-regular-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { LinkingProvider, LinkLibrary } from "./LinkingProvider";
+
+import styles from "./componentStyles/Breadcrumb.module.css";
+import { IndexLink } from "./Links";
+
+interface Crumb {
+  label: string | typeof FontAwesomeIcon;
+  LinkComponent?: (props: any) => React.ReactElement | null; // eslint-disable-line @typescript-eslint/no-explicit-any
+  LinkProps?: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
+}
+
+interface Props {
+  linkLibrary: LinkLibrary;
+  parts: Crumb[];
+}
+
+/**
+ * Navigational bread crumbs defining where the user is at the site.
+ */
+export const BreadCrumb: React.FC<Props> = (props) => {
+  if (!props.parts) {
+    return null;
+  }
+  const parts = [...props.parts];
+  const leaf = parts.pop();
+
+  return (
+    <div className={styles.root}>
+      {parts.map((crumb, i) => (
+        <Fragment key={`${crumb.label}-${i}`}>
+          <MidCrumb linkLibrary={props.linkLibrary} crumb={crumb} />
+          <Separator />
+        </Fragment>
+      ))}
+
+      <LeafCrumb crumb={leaf} />
+    </div>
+  );
+};
+
+export const DefaultHomeCrumb: Crumb = {
+  LinkComponent: IndexLink,
+  label: (
+    <FontAwesomeIcon fixedWidth icon={faHouse} className={styles.separator} />
```

Comment:
Also the typings for `icon` prop seems to be incorrect, here as well as elsewhere.

Reply:
Seems like I needed to typecast them, not sure why those were incorrect in the first place, but should be fixed now 👍

Reply:
See https://github.com/thunderstore-io/thunderstore-ui/pull/572#discussion_r1048117092 (and resolve this conversation).

Also FYI @VilppeRiskidev & @Cyanideankst: in the future ensure fontawesome library versions are in sync.